project.clj

```diff
@@ -39,7 +39,7 @@
                  [org.cyverse/cyverse-groups-client "0.1.7"]
                  [org.cyverse/common-cfg "2.8.1"]
                  [org.cyverse/common-cli "2.8.1"]
-                 [org.cyverse/common-swagger-api "2.10.2"]
+                 [org.cyverse/common-swagger-api "2.10.3-SNAPSHOT"]
```

Comment:
I actually just published version `2.10.3`. This may have to be bumped up to `2.10.4-SNAPSHOT`.

Reply:
Yep 👍 That's what I was kind of getting at in the PR description: this will have to be bumped to `2.10.4` or higher, depending on when other changes are pushed 😉

Reply:
Oh, heh. I probably would have noticed that if I'd actually read the PR description. 😖

Reply:
heh, no problem 👍